src/app/app.component.html

```diff
@@ -1,13 +1,23 @@
-<app-header></app-header>
+<app-header>
+  <div class="header-right-content">
+    <app-button buttonText="Login"></app-button>
+  </div>
+</app-header>
+
 <div class="app-root">
   <!-- Search component -->
   <!--<app-search></app-search>-->
 
-  <!-- Info component -->
-  <!--<app-info></app-info>-->
+  <app-info
+    title="Your List Is Empty"
+    text="Please use 'Add New Course' button to add your first course"
+  >
+    <app-button buttonText="Add new course"></app-button>
```

Comment:
As `buttonText` is an input property in AppButtonComponent, please use the appropriate `[]` notation.